### PR TITLE
Update Beam runner options descriptions

### PR DESCRIPTION
**Pull request by mrshenli**

### CODE REVIEW

File: docs/beamrunner/reference.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ The Streams Runner package contains the following directories:
...
+      <td>A list of JAR files (separated by colons) that are required to run the Apache Beam application. Include the JAR files that contain your program and any dependencies. (You don’t need to include Beam Google IO SDK or core Beam JAR files.) The listed JAR files are added to the SAB file. Globs (wildcards) may be used to specify files (e.g., <code class="highlighter-rouge">foo/bar/\*.jar</code>). However, globs in directory paths are not supported (e.g., <code class="highlighter-rouge">\*\*/\*.jar</code>).<br /><br /><strong>Note</strong>: The use of fat or uber JAR files can reduce the number of JAR files that must be specified, but take care not to include JAR files that are provided by Streams Runner. Including redundant dependencies can increase the application archive and can negatively impact submission times to IBM Cloud.</td>
```
Comment (mrshenli) [inline docs/beamrunner/reference.md]: "\\*" is displayed differently in my local atom editor compared to github. Let me see if I can get it show correctly in both environments.

File: docs/beamrunner/reference.md [hunk 2]
```diff
@@ -71,13 +71,33 @@ The Streams Runner package contains the following directories:
...
+      <td>Set the tracing and logging level of StreamsRunner translation and runtime. If specified, overrides all other tracing options. Levels: <code class="highlighter-rouge">ERROR</code>, <code class="highlighter-rouge">WARN</code>, <code class="highlighter-rouge">INFO</code>, <code class="highlighter-rouge">DEBUG</code>, <code class="highlighter-rouge">TRACE</code></td>
+      <td><code class="highlighter-rouge">[null]</code></td>
+    </tr>
+    <tr>
+      <td><code class="highlighter-rouge">traceTranslation</code></td>
+      <td>Set the translation trace level for the application. Specify a single level: <code class="highlighter-rouge">ERROR</code>, <code class="highlighter-rouge">WARN</code>, <code class="highlighter-rouge">INFO</code>, <code class="highlighter-rouge">DEBUG</code>, <code class="highlighter-rouge">TRACE</code> to set all loggers to level and/or set individual components. Components include Runner, Streams. Example (set all to <code class="highlighter-rouge">INFO</code> except StreamsRunner logging to <code class="highlighter-rouge">WARN</code>):<br /><br /><code class="highlighter-rouge">INFO,Runner=WARN</code></td>
```
Comment (pgerv12) [inline docs/beamrunner/reference.md]: "Components include Runner and Streams. For example, to set Streams Runner to WARN and all other loggers to INFO, specify the following:"

File: docs/beamrunner/reference.md [hunk 3]
```diff
@@ -71,13 +71,33 @@ The Streams Runner package contains the following directories:
...
+      <td><code class="highlighter-rouge">[null]</code></td>
+    </tr>
+    <tr>
+      <td><code class="highlighter-rouge">traceTranslation</code></td>
+      <td>Set the translation trace level for the application. Specify a single level: <code class="highlighter-rouge">ERROR</code>, <code class="highlighter-rouge">WARN</code>, <code class="highlighter-rouge">INFO</code>, <code class="highlighter-rouge">DEBUG</code>, <code class="highlighter-rouge">TRACE</code> to set all loggers to level and/or set individual components. Components include Runner, Streams. Example (set all to <code class="highlighter-rouge">INFO</code> except StreamsRunner logging to <code class="highlighter-rouge">WARN</code>):<br /><br /><code class="highlighter-rouge">INFO,Runner=WARN</code></td>
+      <td><code class="highlighter-rouge">Runner=INFO,Streams=WARN</code></td>
+    </tr>
+    <tr>
+      <td><code class="highlighter-rouge">traceRuntime</code></td>
+      <td>Set the runtime trace level for the application. See traceTranslation for more information.</td>
```
Comment (pgerv12) [inline docs/beamrunner/reference.md]: Should we set `<code>` around traceTranslation here?

File: docs/beamrunner/reference.md [hunk 4]
```diff
@@ -71,13 +71,33 @@ The Streams Runner package contains the following directories:
...
...ass="highlighter-rouge">split</code> API. For <code class="highlighter-rouge">UnboundedSource</code>, the runner uses the specified parallel width as the desiredNumSplits argument in <code class="highlighter-rouge">split</code>. For <code class="highlighter-rouge">BoundedSource</code>, the runner uses the specified parallel width to calculate the <code class="highlighter-rouge">desiredBundleSizeBytes</code>. However, the <code class="highlighter-rouge">split</code> API does not guarantee to respect the desired widths. Some sources might be unsplittable, and some only split to a certain number of sub-sources. Source parallel width is set to either the specified width or the number of sub-sources, whichever is smaller. If there were more sub-sources than specified width, one channel could contain multiple sub-source instances.<br /><br />For <code class="highlighter-rouge">KV</code> <code class="highlighter-rouge">PCollection</code>s, it is guaranteed that the same key is always processed in the same parallel instance, correctly preserving per-key states.<br /><br />Changing parallel width (since IBM Streams release 4.3.0) dynamically at runtime will <b>break</b> Beam pipelines.</td>
```
Comment (pgerv12) [inline docs/beamrunner/reference.md]: Should we mark this _Experimental_ at the top of the description?

File: docs/beamrunner/reference.md [hunk 5]
```diff
@@ -95,9 +115,9 @@ For the full list of pipeline options, enter  `--help=StreamsPipelineOptions` on
 
 | Parameter | Description | Default value |
 | --- | --- | --- |
-| `restUrl` | The URL for the REST API when you use the `DISTRIBUTED` context type. This parameter is required if you implement metrics in the application. | [null] |
-| `userName` | The user name for basic authentication with REST API when you use the `DISTRIBUTED` context type. | [null] |
-| `userPassword` | The user password for basic authentication for REST API when you use the `DISTRIBUTED` context type. | [null] |
+| `restUrl` | The URL for the REST API when you use the `DISTRIBUTED` context type. This parameter is required if you implement metrics in the application. | The return value of command `streamtool geturl` |
+| `userName` | The user name for basic authentication with REST API when you use the `DISTRIBUTED` context type. | The `user.name` system property |
+| `userPassword` | The user password for basic authentication for REST API when you use the `DISTRIBUTED` context type. If the option value is a path of a readable file (recommended) the password used will be read from first line of the file. Otherwise the option value is used as the password. | [null] |
```
Comment (pgerv12) [inline docs/beamrunner/reference.md]: "A path to a file containing the user password (recommended) or a string containing the user password for basic authentication... . If the option value is a path to a readable file, the first line of the file will be used as the password."

**Review by pgerv12 — CHANGES_REQUESTED** [review]
Couple of changes.

**Review by pgerv12 — APPROVED** [review]

**Pull request merged**